x-pack/plugins/security_solution/public/detections/components/rules/related_integrations/use_integration_privileges.ts

```diff
@@ -8,6 +8,12 @@
 import { useKibana } from '../../../../common/lib/kibana';
 import { IntegrationPrivileges } from './integration_privileges';
 
+/**
+ * Hook for determining if user has fleet/integrations/SOM privileges for fetching
+ * installed integrations. Initially used as we weren't using the fleet.internalReadonlySoClient
+ * for fetching integrations, but keeping this around for a release or two as we add more
+ * fleet/integration features within Security Solution in case it needs to be leveraged for those.
+ */
```

Comment:
We can remove this hook, but figured it might be nice to have around for a release or two as we add more integrations features. I could see a feature like surfacing `# of agents assigned this integration policy` being limited by these privileges.

Reply:
Sounds good!